core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-announcements/dot-toolbar-announcements.component.html

```diff
@@ -23,19 +23,21 @@ <h5 class="announcements__title">{{ 'announcements' | dm }}</h5>
 
 <div class="announcements__container">
     <i class="pi pi-external-link"></i>
-    <a class="announcements__link" [href]="linkToDotCms" data-testId="announcement_link">{{
-        'announcements_show_all' | dm
+    <a class="announcements__link" [href]="linkToDotCms()" data-testId="announcement_link">{{
+        'announcements.show.all' | dm
     }}</a>
 </div>
 
 <h5 class="announcements__title">{{ 'announcements_knowledge_center' | dm }}</h5>
 <div class="announcements__about">
-    <a *ngFor="let link of knowledgeCenterLinks" [href]="link.url" target="_blank">{{
-        link.label
-    }}</a>
+    @for (item of knowledgeCenterLinks(); track item.id;) {
+    <a [href]="item.url" target="_blank">{{ item.label }}</a>
```

Comment:
SonarQube probably will throw an error here. You should add `rel="noopener"` to make it pass the SQ test